apps/taxes/src/modules/taxjar/taxjar-configuration.service.ts

```diff
@@ -64,12 +64,13 @@ export class TaxJarConfigurationService {
 
   async post(config: TaxJarConfig): Promise<{ id: string }> {
     this.logger.debug(`.post called with value: ${JSON.stringify(config)}`);
-    const taxJarClient = new TaxJarClient(config);
-    const validation = await taxJarClient.ping();
+    const validationService = new TaxJarValidationService();
+    const validation = await validationService.validate(config);
```

Comment:
if you follow my comment above, you can just do 

`await  validationService.validate(config)` 

and you don't need any "ifs" below which add complexity, it will pass error